assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
 }
 
 
-@odm.model(index=True, store=False, description="Submission Parameters")
+@odm.model(index=True, store=False, description="""Specifies the parameters set at the time of submission, such as whether to perform a deep scan, if the submission should generate an alert, and any specific service selections or settings.
+""")
 class SubmissionParams(odm.Model):
     classification = odm.Classification(default=Classification.UNRESTRICTED,
-                                        description="Original classification of the submission")
-    deep_scan = odm.Boolean(default=False, description="Should a deep scan be performed?")
-    description = odm.Text(store=True, copyto="__text__", description="Description of the submission")
-    generate_alert = odm.Boolean(default=False, description="Should this submission generate an alert?")
-    groups = odm.List(odm.Keyword(), default=[], description="List of groups related to this scan")
-    ignore_cache = odm.Boolean(default=False, description="Ignore the cached service results?")
+                                        description="The initial security classification for the submission, indicating its confidentiality.")
+    deep_scan = odm.Boolean(default=False, description="**TODO**: **Original**:Should a deep scan be performed?  **Generated**:Specifies whether a more intensive scan should be conducted on the submission.")
```

Comment:
We need a better description of deep scan... Looking for others to help, but will start the suggestions with:
`Enables experimental and potentially misleading reports by removing certain safeguards and limitations to find as much information as possible.`

Reply:
Totally agree with you.  I used your new description as extra details to my prompt and got the following description:

> Enables a comprehensive examination of the submission by lifting standard safeguards and constraints, utilizing experimental methods and allowing for the exploration of potentially ambiguous findings to maximize the extraction of information.

What do you think?

Reply:
I agree with this description!